Tema-5/HW-2/style.css

```diff
@@ -0,0 +1,11 @@
+*{
+    box-sizing: border-box;
+    width: 75%;
```

Comment:
Same. Universal selector e folosit in general pentru resetare margins si padding, setare border-box.

Reply:
am modificat si am scos si aici selector-ul universal